web-components/src/components/icons/ProfileIcon.tsx

```diff
@@ -0,0 +1,79 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+import { sxToArray } from '../../utils/mui/sxToArray';
+
+interface ProfileIconProps extends SvgIconProps {}
+
+export default function ProfileIcon({
+  sx,
+  ...props
+}: ProfileIconProps): JSX.Element {
+  return (
+    <SvgIcon
+      sx={[{ color: '#4FB573' }, ...sxToArray(sx)]}
+      width="13"
+      height="13"
+      viewBox="0 0 13 13"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <svg
```

Comment:
You don't have to specify `svg` tag again since it's done by `SvgIcon`.
Also, the with/height/viewport differs between `SvgIcon` and `svg`.
You can check [this component](https://github.com/regen-network/regen-web/blob/dev/web-components/src/components/icons/EmptyCartIcon.tsx) for an up-to-date example.

Reply:
i've removed some cruft from this but i feel like there is still probably some more i could remove...